apps/chat/src/components/Chat/ChangePathDialog.tsx

```diff
@@ -206,7 +209,11 @@ export const ChangePathDialog = ({
         />
       </SelectFolderHeader>
       <SelectFolderFooter
-        handleNewFolder={() => handleAddFolder()}
+        handleNewFolder={() =>
+          handleAddFolder(
+            constructPath(ApiKeys.Files, BucketService.getBucket()),
```

Comment:
Move in `FileService` as static method/property `rootFilePath`

Reply:
I created utility for such thing
Please recheck